config/runtime/tests/rt-tests.jinja2

```diff
@@ -13,7 +13,7 @@
         ARTIFACTORIAL_TOKEN: {{ artifactorial_token|default('') }}
         ARTIFACTORIAL_URL: {{ artifactorial_url|default('') }}
         AFFINTIY: {{ affinity|default('0-1') }}
-        DURATION: {{ duration|default('300s') }}
+        DURATION: {{ duration|default('600s') }}
```

Comment:
@nuclearcat If the timeout is set to 10 minutes and the test duration is also the same, wouldn't it cause the forceful termination of the tests? Should I decrease the duration to a lower value? What value would be best here?

Reply:
I think it should be fine, total job value is safeguard that job doesnt occupy chromebook for too long time, so it should be fine. More comprehensive approach will be estimating if there is any other tasks before and after this timeout, then subtracting their average duration from this 600s.